Source/ASDisplayNode.mm

```diff
@@ -600,13 +600,13 @@ - (BOOL)isNodeLoaded
 - (BOOL)_locked_isNodeLoaded
 {
   ASAssertLocked(__instanceLock__);
-  return (_view != nil || (_layer != nil && _flags.layerBacked));
+  return _ASIsNodeLoaded(self);
 }
 
-/// Called without the lock. Client is responsible for locking safety.
-- (BOOL)_unsafe_unlocked_isNodeLoaded
+/// Can be called without the node's lock. Client is responsible for thread safety.
+ASDISPLAYNODE_INLINE AS_WARN_UNUSED_RESULT BOOL _ASIsNodeLoaded(ASDisplayNode *node)
 {
-  return (_view != nil || (_layer != nil && _flags.layerBacked));
+  return (node->_view != nil || (node->_layer != nil && node->_flags.layerBacked));
```

Comment:
What if we get rid of this function and just put `_layer != nil` inside the places where we need it?